examples/pytorch/graphsage/train_sampling_unsupervised.py

```diff
@@ -298,7 +298,7 @@ def main(args, devices):
     val_mask = g.ndata['val_mask']
     test_mask = g.ndata['test_mask']
     g.ndata['features'] = features
-    g.create_format_()
+    g.create_formats_()
```

Comment:
I'm surprised that there are so many examples using this API. The API should be used quite rarely. Even for mini-batch training, whether materializing all formats in the data loading process is beneficial is actually not clear. I feel we should avoid using this API in the examples unless absolutely sure. cc @classicsong for opinion.